.github/actions/create-jira-issue/entrypoint.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+sprint=$(curl -u ${JIRA_AUTH} -X GET -H "Content-Type: application/json" https://scalar-labs.atlassian.net/rest/agile/1.0/board/1/sprint?state=active | jq '.values[].id')
```

Comment:
```suggestion
sprint_id=$(curl -u ${JIRA_AUTH} -X GET -H "Content-Type: application/json" https://scalar-labs.atlassian.net/rest/agile/1.0/board/1/sprint?state=active | jq '.values[].id')
```

Reply:
Also need to add `jq` to `Dockerfile` below.

https://github.com/scalar-labs/scalar-terratest/blob/6b66cf473007e6f2de00dc5ac6564b4a9120e940/.github/actions/create-jira-issue/Dockerfile#L3

Reply:
Thank you. Fixed in d390d51